eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway.php

```diff
@@ -15,177 +15,139 @@
 use eZ\Publish\Core\Persistence\Legacy\Content\StorageFieldDefinition;
 
 /**
- * Base class for content type gateways.
+ * Content Type Gateway.
```

Comment:
Looks a little bit redundant as this is `eZ\Publish\Core\Persistence\Legacy\Content\Type\Gateway`

Reply:
> Looks a little bit redundant as this is `eZ\Publish\Core\Persistence\Legacy\Content\Type\Gateway`

TBH I don't like the direction "self commenting" is heading. Verbose information is still more pleasant and moreover faster to read.

![image](https://user-images.githubusercontent.com/7099219/76352726-ce003200-630f-11ea-8b0b-ba6053e7bf3d.png)

How about we keep PHPDocs which provide more information than the used name (so Developers won't have to infer from the namespace)?

Reply:
I'm a big fan of the description in docblock, no matter at what level (class, method, properties). But I am against those that are redundant. I opt for providing more information. But this is really not a big deal. I think we should proceed.